### ISSUE TITLE
n for DE p-value adjustment

### ISSUE THREAD

**Issue by lazappi** [satijalab/seurat#905]
In `FindMarkers` when performing multiple testing correction the number of genes in the dataset rather than the number of genes that were tested is used for n.

https://github.com/satijalab/seurat/blob/9fae278a495818a5c3358fbe34b957d950416e77/R/differential_expression.R#L304

What is the rationale behind that? Intuitively I would have thought using the number of tested genes makes sense but perhaps there is a reason for using the total number of genes?

**Comment by mschilli87** [comment]
@satijalab: I agree with your default choice but I think a new parameter that enables using the smaller set for testing (only for those who know what they do, default `FALSE`) would have some use cases (think of mixed species experiments most cells from one species and few cells from many species that are excluded from selected analyses etc.).
